wee_slack.py

```diff
@@ -2579,7 +2577,7 @@ def get_members(self):
             # complete, so we have to fetch members for all channels.
             s = SlackRequest(
                 self.team,
-                "conversations.members",
+                "membership",
```

Comment:
Is `membership` a method that works in your Slack workspace? I just get `unknown_method` from it. You also haven't handled the response anywhere, so making this request won't have any effect.